meson.build

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: LGPL-2.1 OR BSD-2-Clause
+project('libbpf', 'c',
+        version : '0.0.2',
```

Comment:
there's another PR that is changing this to 0.0.3 (as it is in the kernel)

Reply:
Resolved, thanks!